### PR TITLE
Daily print view header

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
Daily print view header is ready for review.  To be reviewed alongside the clintb/daily-print-view branch in the `blip` repo.

@jebeck I ended up not needing the image buffer preloading, as the webpack url-loader plugin worked for loading images within the browser.

I added fixtures for 2 patient account types (standard and fakeChildAccount) and added a storybook to test the fakeChildAccount one as well.

### CODE REVIEW

**Comment by cbwebdevelopment** [comment]
[<L> [blip] Daily print view first production release](https://trello.com/c/tjD655oz/182-l-blip-daily-print-view-first-production-release)

**Comment by cbwebdevelopment** [comment]
[<L> [blip] Daily print view first production release](https://trello.com/c/tjD655oz/182-l-blip-daily-print-view-first-production-release)

**Comment by jebeck** [comment]
And here too the commit log in this PR is way more than your changes, so I've updated from master and now there are conflicts... Handle them @cbwebdevelopment when you get the chance?

This is my bad for not taking a closer look at those branches before instructing you to branch off *and* update from master, but it's also probably a good lesson on how things can go wrong (i.e., out of date) when you have a long-running branch and lots of changes coming into master and you try to do fancy branching strategies.

**Comment by cbwebdevelopment** [comment]
Conflicts resolved :)

File: src/modules/print/DailyPrintView.js [hunk 1]
```diff
@@ -345,6 +357,92 @@ class DailyPrintView {
...
+    // Calculate the remaining available width so we can
+    // center the print text between the patient/title text and the logo
+    const availableWidth = this.doc.page.width - _.reduce([
+      this.patientInfoBox.width,
+      this.dividerWidth,
+      this.titleWidth,
+      this.logoWidth,
+      this.margins.left,
+      this.margins.right,
+    ], (a, b) => a + b, 0);
```
Comment (jebeck) [inline src/modules/print/DailyPrintView.js]: Very minor point, but does our ESLint config enforce `(a, b) => a + b` here? I'd prefer `(a, b) => (a + b)` personally, but I'm happy to stick with what the Airbnb config enforces if it enforces how you have it.

File: src/utils/datetime.js [hunk 2]
```diff
@@ -96,6 +96,14 @@ export function formatClocktimeFromMsPer24(milliseconds, format = 'h:mm a') {
 }
 
 /**
+ * formatCurrentDate
```
Comment (jebeck) [inline src/utils/datetime.js]: Take a look at the README inside utils/ and try to follow the instructions for generating a fresh copy of utils API docs (based on the JSDoc comments). Then review [the info about working on docs w/GitBook](http://developer.tidepool.io/docs/docs/workflow.html) and easy have a go at publishing your revised docs, or we can chat about it and review before you try.

File: src/modules/print/DailyPrintView.js (hunk 1 above)
Comment (cbwebdevelopment) [inline src/modules/print/DailyPrintView.js]: I agree.  Will do.  I doubt the airbnb config enforces it either way.

**Pull request merged**